mapie/conformity_scores.py

```diff
@@ -183,7 +187,7 @@ def get_signed_conformity_scores(
         return np.subtract(y, y_pred)
```

Comment:
Why not return np.abs(np.subtract(y, y_pred))?

Reply:
Notably because Absolute is explicit in the class' name

Reply:
The conformity score is computed in two steps : 
- Compute the signed conformity score with `get_signed_conformity_scores`
- Compute the "final" conformity score with `get_conformity_scores`, consisting in the absolute value if the `sym` argument is set to True, and the signed conformity score if `sym=False` (`GammaConformityScore` for instance)

`MapieRegressor` distinguishes the two ways of computing the resulting lower/upper bounds in lines 684-691.